Mathlib/Data/Seq/Computation.lean

```diff
@@ -1076,7 +1076,7 @@ theorem LiftRel.trans (R : α → α → Prop) (H : Transitive R) : Transitive (
 #align computation.lift_rel.trans Computation.LiftRel.trans
 
 theorem LiftRel.equiv (R : α → α → Prop) : Equivalence R → Equivalence (LiftRel R)
-  -- Porting note: The code below was:
+    -- Porting note: The code below was:
```

Comment:
```suggestion
  -- Porting note: The code below was:
```

Reply:
same as above. Unindented and moved below